### ISSUE TITLE
Empty `<text/>` on lyric tag imports as .text = None and causes TypeError on access

### ISSUE THREAD

**Issue by jacobtylerwalls** [cuthbertLab/music21#1036]
**music21 version**

dev

**Problem summary**
When trying to horizontally align rows of lyrics/annotations, it's helpful to have an empty value for a dummy lyric. I've resorted to using `' '`, but I keep accidentally using `''` and causing TypeErrors. `None` is no good because `.addLyric()` turns `None` -> `"None"`.

**Steps to reproduce**

```
>>> n = note.Note()
>>> n.addLyric('')  # want to insert a dummy lyric somehow
>>> n.lyrics  # so far so good
[<music21.note.Lyric number=1 syllabic=single text=''>]
>>> out = n.write()  # writes empty <text/> tag
>>> inn = converter.parse(out)  # imports as .text = None
>>> inn.recurse().notes[0].lyrics
[<music21.note.Lyric number=1 identifier='1' syllabic=single>]
>>> inn.recurse().notes[0].lyric  # can't handle .text = None
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/music21/note.py", line 587, in _getLyric
    return '\n'.join(allText)
TypeError: sequence item 0: expected str instance, NoneType found
```

**Suggested fix**
We should decide if `None` or `''` represents the empty value and then fix the importer or possibly `_getLyric()`.

**More information**
It might be possible I'm supposed to be aligning lyrics with the number attribute instead of trying to insert dummy empty lyrics. Not certain.

**Comment by jacobtylerwalls** [comment]
Crash fixed in #1053

**Comment by jacobtylerwalls** [comment]
Agree with Michael that we can live with two values for empty lyrics for now, as there is just as much ambiguity in musicxml space as music21 space (and insisting on one or the other might catch some folks by surprise): https://github.com/cuthbertLab/music21/pull/1053#discussion_r663293971
